src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -72,7 +72,7 @@ public static function register($mode = 0)
             }
 
             $mode = $getMode();
-            $trace = debug_backtrace(true);
+            $trace = debug_backtrace(\defined(DEBUG_BACKTRACE_PROVIDE_OBJECT) ? DEBUG_BACKTRACE_PROVIDE_OBJECT : true); // simplify when dropping php < 5.3.6
```

Comment:
Note that `DEBUG_BACKTRACE_PROVIDE_OBJECT` is the default value for this argument.